ui/components/multichain/connected-site-menu/connected-site-menu.js

```diff
@@ -90,7 +90,7 @@ export const ConnectedSiteMenu = ({
                   ? BorderColor.successDefault
                   : BackgroundColor.backgroundDefault
               }
-              borderWidth={isConnectedtoOtherAccountOrSnap ? 2 : 3}
+              borderWidth={2}
```

Comment:
Before (not connected):
<img width="358" alt="Screenshot 2024-12-18 at 15 41 03" src="https://github.com/user-attachments/assets/19bbe965-1573-4f66-bed4-4150c2a9fb0c" />

After (not connected):
<img width="353" alt="Screenshot 2024-12-18 at 15 39 32" src="https://github.com/user-attachments/assets/adf7230a-80c1-48e7-947d-d7d8dd846b84" />

Before (connected):
<img width="355" alt="Screenshot 2024-12-18 at 15 41 21" src="https://github.com/user-attachments/assets/305ccb6d-f721-4c3b-b46f-ff0e7306826c" />

After (connected)
<img width="359" alt="Screenshot 2024-12-18 at 15 39 17" src="https://github.com/user-attachments/assets/444c9708-8486-4ff8-8dd0-f15b40f713cb" />
: